mpp/shmemx_c_func.h4

```diff
@@ -166,3 +166,70 @@ define(`SHMEM_C_CTX_PUT_N_SIGNAL_NBI',
 SHMEM_DECLARE_FOR_SIZES(`SHMEM_C_CTX_PUT_N_SIGNAL_NBI')
 SHMEM_C_CTX_PUT_N_SIGNAL_NBI(mem,1);
 
+
+/* Team Management Routines */
+SHMEM_FUNCTION_ATTRIBUTES int SHPRE()shmemx_team_my_pe(shmemx_team_t team);
+
+SHMEM_FUNCTION_ATTRIBUTES int SHPRE()shmemx_team_n_pes(shmemx_team_t team);
+
+SHMEM_FUNCTION_ATTRIBUTES void SHPRE()shmemx_team_get_config(shmemx_team_t team, shmemx_team_config_t *config);
+
+SHMEM_FUNCTION_ATTRIBUTES int SHPRE()shmemx_team_translate_pe(shmemx_team_t src_team, int src_pe, shmemx_team_t dest_team);
+
+SHMEM_FUNCTION_ATTRIBUTES int SHPRE()shmemx_team_split_strided(shmemx_team_t parent_team, int PE_start, int PE_stride, int PE_size, shmemx_team_config_t *config, long config_mask, shmemx_team_t *new_team);
+
+SHMEM_FUNCTION_ATTRIBUTES int SHPRE()shmemx_team_split_2d(shmemx_team_t parent_team, int xrange, shmemx_team_config_t *xaxis_config, long xaxis_mask, shmemx_team_t *xaxis_team, shmemx_team_config_t *yaxis_config, long yaxis_mask, shmemx_team_t *yaxis_team);
+
+SHMEM_FUNCTION_ATTRIBUTES int SHPRE()shmemx_team_destroy(shmemx_team_t team);
+
+SHMEM_FUNCTION_ATTRIBUTES int SHPRE()shmemx_team_create_ctx(shmemx_team_t team, long options, shmem_ctx_t *ctx);
+
+SHMEM_FUNCTION_ATTRIBUTES int SHPRE()shmemx_ctx_get_team(shmem_ctx_t ctx, shmemx_team_t *team);
+
+SHMEM_FUNCTION_ATTRIBUTES int SHPRE()shmemx_sync(shmemx_team_t team);
+SHMEM_FUNCTION_ATTRIBUTES int SHPRE()shmemx_team_sync(shmemx_team_t team);
+
+/* Team Collective Routines */
+define(`SHMEM_C_BCAST',
+`SHMEM_FUNCTION_ATTRIBUTES int SHPRE()shmemx_$1_broadcast(shmemx_team_t team, $2 *dest, const $2 *source, size_t nelems, int PE_root)')dnl
+SHMEM_DECLARE_FOR_COLLECTIVES(`SHMEM_C_BCAST')
+
```

Comment:
This newline will generate an extra newline in the output header.  Please review the generated header for formatting.